src/AmbientSounds.Uwp/Strings/cs-CZ/Resources.resw

```diff
@@ -175,16 +280,16 @@
     <value>Děkujeme</value>
   </data>
   <data name="SettingsThemeHeader.Text" xml:space="preserve">
-    <value>Téma (újraindítást igényel)</value>
+    <value>Motiv aplikace</value>
   </data>
   <data name="SettingsThemeDefaultRadio.Content" xml:space="preserve">
-    <value>Automatikus</value>
+    <value>Systémový</value>
   </data>
   <data name="SettingsThemeDarkRadio.Content" xml:space="preserve">
-    <value>Sötét</value>
+    <value>Tmavý</value>
   </data>
   <data name="SettingsThemeLightRadio.Content" xml:space="preserve">
-    <value>Fény</value>
+    <value>Světlý</value>
```

Comment:
@MichalMoudry thank you again for your help with this! We've changed how translations work. You no longer have to edit the .resw file. Instead, we'll be editing the .xlf file. All edits made in the .xlf will be automatically added into the .resw when you compile the app. This new system is more flexible than editing the .resw files directly. Please try reverting these changes > then applying the translation fix in the appropriate .xlf. More details here: https://github.com/jenius-apps/ambie#improving-an-existing-language-can-be-done-with-any-text-editor